src/components/NavBar.tsx

```diff
@@ -0,0 +1,30 @@
+import { NavLink } from 'react-router-dom';
+
+export const Navbar = () => {
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            to="/"
+            className={({ isActive }) => `navbar-item ${isActive ? 'has-background-grey-lighter' : ''}`}
```

Comment:
You don't need here ternary operator. If isActive will be false, the class will be not added